editoast/src/views/path/pathfinding.rs

```diff
@@ -363,7 +369,7 @@ pub async fn pathfinding_from_train_batch(
     core: Arc<CoreClient>,
     infra: &Infra,
     train_schedules: &[TrainSchedule],
-    rolling_stocks: &HashMap<String, RollingStockModel>,
+    consists: &HashMap<String, PhysicsConsistParameters>,
```

Comment:
Can this function take a `consists: &[PhysicsConsistParameters]`? That'd be more consistent with the rest of the signature and it would avoid unnecessary clones. A local hash map only containing references can be built internally.